<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentFiles</class>
 <widget class="QWidget" name="RecentFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="messageWidget">
     <property name="text">
      <string>Cleared the activity history.</string>
     </property>
     <property name="messageType">
      <enum>KMessageWidget::Positive</enum>
     </property>
     <property name="icon">
      <iconset theme="dialog-ok-apply">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelKeepHistory">
       <property name="text">
        <string>Keep history:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="kcfg_keepHistoryFor"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClearRecentHistory">
         <property name="text">
          <string>Clear History</string>
         </property>
         <property name="icon">
          <iconset theme="edit-clear-history">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="kcfg_whatToRemember">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioRememberAllApplications">
          <property name="text">
           <string>For a&amp;ll applications</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioDontRememberApplications">
          <property name="text">
           <string>&amp;Do not remember</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRememberSpecificApplications">
          <property name="text">
           <string>O&amp;nly for specific applications:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRememberDocuments">
       <property name="text">
        <string>Remember opened documents:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QQuickWidget" name="viewBlacklistedApplications">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="resizeMode">
      <enum>QQuickWidget::SizeRootObjectToView</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="kcfg_blockedByDefault">
       <property name="text">
        <string>Blacklist applications not on the list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioRememberSpecificApplications</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_blockedByDefault</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
